autoload/db_ui/dbout.vim

```diff
@@ -8,7 +8,8 @@ function! db_ui#dbout#jump_to_foreign_table() abort
   let cell_line_number = s:get_cell_line_number(scheme)
   let cell_range = s:get_cell_range(cell_line_number, getcurpos(), parsed.scheme)
 
-  let field_name = trim(getline(cell_line_number - 1)[cell_range.virtual.from : cell_range.virtual.to])
+  let field_name = trim(getline(cell_line_number - 1)[get(cell_range, 'virtual', cell_range).from : get(cell_range, 'virtual', cell_range).to])
```

Comment:
I'm using `get()` here in a way that I'm hoping will be clever. The thinking is, if `s:get_cell_range()` has implicitly returned `s:get_virtual_cell_range()`, then `get()` will match the `'virtual'` key in the `cell_range` dict.

If it didn't, then `get()` will select the entire `cell_range` dict as a default instead.


I've tested it on Postres and MariaDB, and this has restored functionality while maintaining the ability to change behavior away from the callsite.

Reply:
That's ok, just please either extract it to variable od do some line breaks. It's a bit too long.